### PR TITLE
[Hero][Jed] Mobile to Tablet Screens

### PR DESCRIPTION
**Pull request by jedmamosto**
Finished the mobile to tablet screen sizes. Desktop screens will figure out pa. 

Let me know for any formatting, tailwind errors, or file structure errors.

Clickup link: https://app.clickup.com/t/865cm29hf

### CODE REVIEW

**Comment by jedmamosto** [comment]
Added a fix for a missed space.

Also, here are the images:

**Mobile Screen:** 

image

**Tablet Screen:**

image

**Comment by jedmamosto** [comment]
Also added the colors pala to our tailwind config, plus i put the images sa public folder

File: app/components/PrimaryButton.tsx [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+interface PrimaryButtonProps {
+  className?: string;
+  children?: React.ReactNode;
```
Comment (ezerssss) [inline app/components/PrimaryButton.tsx]: `children?: JSX.Element | JSX.Element[]`

File: app/components/PrimaryButton.tsx [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+	return (
+		<button
+			className={`h-10 w-40 rounded-md bg-secondary-100 text-xs font-semibold text-primary-400 sm:h-12 sm:w-52 sm:text-base ${className}`}
+		>
+			{children}
+		</button>
+	);
+}
+
+export default PrimaryButton;
```
Comment (ezerssss) [inline app/components/PrimaryButton.tsx]: looks like unnecessary abstraction, its just a `button` na HTML component, we can just put this directly sa component.

just remove this and put the button directly sa component ur using it in

**Review by ezerssss — CHANGES_REQUESTED** [review]
also compress the images, we want to make the images load faster

**Review by ezerssss — CHANGES_REQUESTED** [review]
doesnt look like the figma for desktop screens

Actual:
image

Figma: 
image

File: app/Hero.tsx [hunk 3]
```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import PrimaryButton from './components/PrimaryButton';
+import Image from 'next/image';
+
+function Hero() {
+	return (
+		<div className="flex flex-col items-center">
```
Comment (ezerssss) [inline app/Hero.tsx]: make this a `section` element

**Review by ezerssss — APPROVED** [review]
lgtm

**Pull request merged**